dist/index.html

```diff
@@ -2,14 +2,233 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Practice</title>
+    <title>DEV.THEYMADE</title>
+    <link rel="stylesheet" href="/style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Aleo:ital,wght@0,400;0,700;1,400&family=Inter&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
     <header>
-      <h1>Hello, World</h1>
+      <div class="brand">
+        <div class="logo">
+          <img src="/images/main_logo.png" alt="Companie's Logo" />
+        </div>
+        <div>
+          <p>Professional web development since 2010</p>
+        </div>
+      </div>
+      <article>
+        <p class="intro">
+          Hello! We are a professional web development team with vast experience
+          in complex projects. We collaborate with design studios from United
+          States, Europe and Asia. If you are here, it means we were recommended
+          to you.
+        </p>
+        <p class="stack">
+          <b>Stack: </b>
+          Redux, Redux-Saga, Webpack, SCSS, Jest, Typescript, Node.js, Nest.js,
+          Express.js, React, Mobx, Tree.js, Canvas, Microservice architecture,
+          Fat client, 3D, 2D
+        </p>
+      </article>
     </header>
-    <script src="main.js"></script>
+    <section class="experience">
+      <h2>Our experience and works</h2>
+      <div class="wrapper">
+        <div class="case-study">
+          <a href="/info/caveat" class="case-study-thumbnail" id="blue">
```

Comment:
if tag has more then one attribute - best practise: one line for one attribute
it helps us more comfortable use `git blame`

Reply:
got it